pyproject.toml

```
@@ -0,0 +1,37 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "JupyterDaskOnSLURM"
```

Let's maybe drop the capitals, that I think are discouraged for Python packages?

```suggestion
name = "jupyter-dask-on-slurm"
```